frontend/src/scenes/onboarding/OnboardingInviteTeammates.tsx

```diff
@@ -0,0 +1,136 @@
+import { LemonBanner, LemonButton, LemonDialog, LemonTextArea, Link } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { CopyToClipboardInline } from 'lib/components/CopyToClipboard'
+import { IconDelete, IconPlus } from 'lib/lemon-ui/icons'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+import { inviteLogic } from 'scenes/settings/organization/inviteLogic'
+import { InviteRow, MAX_INVITES_AT_ONCE } from 'scenes/settings/organization/InviteModal'
+import { userLogic } from 'scenes/userLogic'
+
+import { OrganizationInviteType } from '~/types'
+
+import { OnboardingStepKey } from './onboardingLogic'
+import { OnboardingStep } from './OnboardingStep'
+
+export const OnboardingInviteTeammates = ({ stepKey }: { stepKey: OnboardingStepKey }): JSX.Element => {
+    const { user } = useValues(userLogic)
+    const { preflight } = useValues(preflightLogic)
+    const { invitesToSend, invites } = useValues(inviteLogic)
+    const { appendInviteRow, deleteInvite, updateMessage } = useActions(inviteLogic)
+
+    const invitesReversed = invites.slice().reverse()
+    const areInvitesCreatable = invitesToSend.length + 1 < MAX_INVITES_AT_ONCE
+    const areInvitesDeletable = invitesToSend.length > 1
+
+    return (
+        <OnboardingStep title={`Invite teammates to ${user?.organization?.name || 'PostHog'}`} stepKey={stepKey}>
+            {preflight?.email_service_available ? (
```

Comment:
This is mainly copied from the `InviteModal.tsx`. I should probably extract more things into subcomponents / constant so it's easier to keep these modules in sync.

Reply:
Yeah it seems like it'd be good to extract things so it's not a bunch of duplicate code.